src/lava/lib/optimization/solvers/generic/solver.py

```diff
@@ -17,9 +18,13 @@
 from lava.magma.core.sync.protocol import AbstractSyncProtocol
 from lava.magma.core.sync.protocols.loihi_protocol import LoihiProtocol
 from lava.proc.dense.models import PyDenseModelFloat
+from lava.proc.dense.ncmodels import NcModelDense

```

Comment:
In the public lava-optimization, we should not import ncmodels from the private collab repo!

Reply:
Resolved, thanks